ext/native_test.go

```diff
@@ -750,7 +816,7 @@ func TestNativeTypesWithOptional(t *testing.T) {
 }
 
 func TestNativeTypeConvertToType(t *testing.T) {
-	nt, err := newNativeType(true, reflect.TypeOf(&TestAllTypes{}))
+	nt, err := newNativeType("cel", reflect.TypeOf(&TestAllTypes{}))
```

Comment:
Would you mind making these parameterized tests for the known tag types: `cel`, `json`?

Reply:
Done